src/site/_includes/components/cookie/cookie.ts

```diff
@@ -0,0 +1,41 @@
+export function getCookie(name: string) {
+  const nameEQ = `${name}=`
+  const cookies = document.cookie.split('; ')
+  for (let i = 0, len = cookies.length; i < len; i += 1) {
+    const cookie = cookies[i]
+    if (cookie.indexOf(nameEQ) === 0) {
+      return decodeURIComponent(cookie.substring(nameEQ.length))
+    }
+  }
+  return null
+}
+
+/**
+ * Set the cookie
+ * @param name - The name of the cookie
+ * @param value - The value of the cookie
+ * @param ttl - The time to live in days
+ */
+export function setCookie(name: string, value: string, ttl: number) {
```

Comment:
Might be nice to add 'days' to the `ttl` arg name rather than relying on the JSDoc comment e.g. `ttlDays`, `daysValid`)